zanata-war/src/main/webapp/WEB-INF/pages.xml

```diff
@@ -172,62 +43,16 @@
     </navigation>
   </page>
 
-
   <!-- Admin -->
   <!-- TODO [CDI] find a way to secure a url pattern in CDI -->
   <page view-id="/admin/*" action="#{identity.checkRole('admin')}"></page>
```

Comment:
I have no problem with adding the check to every admin xhtml page; in fact that's the approach we are taking with all other pages.
In fact I prefer that to a servlet filter as the it's clear what actions are being performed by looking at the view.